src/components/svg-canvas-graph.tsx

```diff
@@ -48,6 +54,12 @@ const SvgCanvas = () => {
     const handlePointerDown = useEvent((node: StnId | MiscNodeId, e: React.PointerEvent<SVGElement>) => {
         e.stopPropagation();
 
+        if (mode === 'select') {
```

Comment:
Do we want to support the case where users click select and then start from any nodes? This `handlePointerDown` only works on nodes IMO.

Reply:
It will be banned.